packages/material-ui/src/ClickAwayListener/ClickAwayListener.test.js

```diff
@@ -32,17 +31,23 @@ describe('<ClickAwayListener />', () => {
   });
 
   it('should render the children', () => {
-    const children = <span>Hello</span>;
-    wrapper = mount(<ClickAwayListener onClickAway={() => {}}>{children}</ClickAwayListener>);
+    const ref = React.createRef();
+    const children = <span ref={ref}>Hello</span>;
+    wrapper = mount(
+      <ClickAwayListener getTargetEl={() => ref.current} onClickAway={() => {}}>
```

Comment:
legacy

Reply:
It should have failed the CI. What do you think of removing the spread to EventListener and to add `exactProp()`?
It would be a breaking change: #15126.

Reply:
It would decouple use from the specific implementation which helps removing the `EventListener` (it's currently part of the public API). There may have been users that override e.g. `target`. 

I don't see the need for it now (considering #15126 uses spread props).